defaults/main.yml

```diff
@@ -5,6 +5,8 @@ database_name: opencast
 database_user_name: opencast
 # mariadb root user
 database_root_user: root
+# mariadb root user password
+database_root_password: '4567'
```

Comment:
There should be an explanation that this is now only being used for authentication and that this password is not set

Reply:
I had already adapted the README but I have added additional context in `defaults/main.yml` file now